swap_meet/item.py

```diff
@@ -1,2 +1,19 @@
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, category = "", condition = 0):
+        self.category = category
+        self.condition = condition
+    
+    def __str__(self):
+        return "Hello World!"
+    
+    def condition_description(self):
```

Comment:
👍 

I like that you have a fallback case for conditions greater than 5, and I _love_ that this is a data-driven approach. Now what would you do for conditions less than 0?

Also, while there was no test printing the description for a fractional condition, recall that there _was_ a test that made items with conditions of 3.5. So we should make sure we can handle that situation in our code.

What would happen here if the condition were 3.5? How might we make the code more robust?